### DISCUSSION TITLE
I don't know how to using calculator.. plz help

### DISCUSSION THREAD

**Discussion by yoojinlee-hub** [community/community#78086] [New to GitHub] [answered]
We will use a 6-person team, the total storage required is 1TB, and 10G bandwidth is required. Please let me know exactly the expected amount at this time. <br> And other optional functions are not wanted. <br> In this case, How can I use 'Pricing calaulator(https://github.com/pricing/calculator?feature=packages)'? And what is expected price?

**Comment by erickdc7** [comment, marked as answer]
To use the [GitHub Pricing Calculator](https://github.com/pricing/calculator?feature=packages) for a 6-person team with 1TB of storage and 10G of bandwidth, you'll need to follow these steps:

1. **Navigate to the Calculator**: Go to the [GitHub Pricing Calculator](https://github.com/pricing/calculator?feature=packages) page.
2. **Input Team Size**: Enter "6" for the number of users in your team.
3. **Select Storage and Bandwidth**: Input "1TB" for storage and "10G" for bandwidth under the relevant sections.
4. **Exclude Optional Features**: Since you don't want other optional functions, ensure you do not select any additional features that are not required.
5. **Calculate Price**: The calculator will automatically update the cost estimate based on the inputs you provide.

The expected price will depend on the current rates for GitHub services. For accurate pricing, it's best to use the calculator directly as it will provide the most up-to-date cost based on your specific requirements. Remember, prices are subject to change, so always refer to the official GitHub Pricing page for the latest information.

**Comment by smartcoin222** [comment]
how to calculate the cvv code from this card number 374490036035005?
![photo_2023-12-18_20-23-37](https://github.com/community/community/assets/154286964/52055510-7360-4dab-aeaf-fb52742df385)

**Comment by jameswrightbrown-prog** [comment]
Hey! If you’re not familiar with the calculator, don’t worry it’s pretty straightforward once you know the basics. Since you only need storage and bandwidth, you can focus on those sections in the pricing calculator. Just enter 1TB for storage and 10G for bandwidth, and leave all the optional features unchecked. The calculator should then give you the estimated monthly price based on just those requirements.
 I’d suggest double-checking that you’re selecting the right units (TB vs GB, Gbps vs MBps) so the estimate is accurate. That should give you a clear idea of the expected cost without including any extra features.
